the-list-ui/src/api.js

```diff
@@ -6,15 +6,15 @@ export async function getStudents() {
 }
 
 export async function deleteStudents(id) {
-        await fetch(`${BE_URL}/students/${id}`, {
-        method: 'DELETE',
-        headers: {
-          'Content-Type': 'application/json',
-          'API-Key': 'secret'
-        },
-        body: JSON.stringify({ _id: id})
-    });
-  }
+  await fetch(`${BE_URL}/students/${id}`, {
+    method: 'DELETE',
+    headers: {
+      'Content-Type': 'application/json',
+      'API-Key': 'secret'
+    },
+    body: JSON.stringify({ _id: id })
```

Comment:
Я увидел, что body не используется на сервере.
```suggestion
    }
```